icons/procreate.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Procreate</title><path d="m11.33 15.84-1.35.35-.31.07c-2.71.64-4.81 1.16-6.39 2.2l.1-.1a7.32 7.32 0 0 1 3.2-1.94c.94-.25 2.19-.35 3.52-.47l1.24-.11m2.11-6.72c-.82.21-1.71.42-2.64.69a16.54 16.54 0 0 0-6.14 3.09 12.37 12.37 0 0 0-3.68 5.47 14.97 14.97 0 0 1 9.81-7.96c.93-.42 1.82-.86 2.68-1.31m5.36 3.3c-1.7.83-3.58 1.15-5.53 1.48-2.15.37-4.36.74-6.51 1.84l-.4.23.08-.02c1.9-.5 4.79-.44 7.63-.97h.03a17.7 17.7 0 0 0 4.71-2.55m3.81-9.06a11.66 11.66 0 0 1-3.27 3.15c-.88.61-1.85 1.09-2.96 1.52a43.57 43.57 0 0 1-3.51 1.94c1.92-.39 5.51-1.23 7.28-2.72a8.16 8.16 0 0 0 2.46-3.9m-.71 5.82c-.25.36-.54.71-.86 1.04-3.79 3.87-9.27 2.42-14.51 5.08A13.9 13.9 0 0 0 3.08 18a11.03 11.03 0 0 0-2.94 5.31v.06c-.04.16-.06.33-.08.49v.13H.05L.1 23a14.67 14.67 0 0 1 12.12-12.38c2.26-.43 6.29-1.29 8.3-2.98 2-1.68 3.13-4.56 3.14-7.6V0c.05.35.7 5.58-1.72 9.18Z" style="fill:#fff;stroke-width:0"/></svg>
```

Comment:
I did not review the icon yet, but the linter is failing due to the extra attribute `style`. We need to remove it:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Procreate</title><path d="m11.33 15.84-1.35.35-.31.07c-2.71.64-4.81 1.16-6.39 2.2l.1-.1a7.32 7.32 0 0 1 3.2-1.94c.94-.25 2.19-.35 3.52-.47l1.24-.11m2.11-6.72c-.82.21-1.71.42-2.64.69a16.54 16.54 0 0 0-6.14 3.09 12.37 12.37 0 0 0-3.68 5.47 14.97 14.97 0 0 1 9.81-7.96c.93-.42 1.82-.86 2.68-1.31m5.36 3.3c-1.7.83-3.58 1.15-5.53 1.48-2.15.37-4.36.74-6.51 1.84l-.4.23.08-.02c1.9-.5 4.79-.44 7.63-.97h.03a17.7 17.7 0 0 0 4.71-2.55m3.81-9.06a11.66 11.66 0 0 1-3.27 3.15c-.88.61-1.85 1.09-2.96 1.52a43.57 43.57 0 0 1-3.51 1.94c1.92-.39 5.51-1.23 7.28-2.72a8.16 8.16 0 0 0 2.46-3.9m-.71 5.82c-.25.36-.54.71-.86 1.04-3.79 3.87-9.27 2.42-14.51 5.08A13.9 13.9 0 0 0 3.08 18a11.03 11.03 0 0 0-2.94 5.31v.06c-.04.16-.06.33-.08.49v.13H.05L.1 23a14.67 14.67 0 0 1 12.12-12.38c2.26-.43 6.29-1.29 8.3-2.98 2-1.68 3.13-4.56 3.14-7.6V0c.05.35.7 5.58-1.72 9.18Z"/></svg>
```